src/cdk/drag-drop/directives/drag.ts

```diff
@@ -209,6 +215,13 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
     this._dragRef.reset();
   }
 
+  /**
+   * Gets the current position in pixels the draggable outside of a drop container.
+   */
+  getCurrentStandalonePosition(): {readonly x: number, readonly y: number} {
```

Comment:
I put in `standalone` to emphasize that this only applies to a `CdkDrag` that's outside of a `CdkDragList`.

Reply:
I think it would make sense to document that `setPosition` only applies when the draggable isn't embedded in a drop-list. The getPosition would just always work.

Reply:
`getPosition` won't work either, because we only apply a transform while the user is moving it, when it's inside a drop list.

Reply:
What do you think about `getFreeDragPosition` and `setFreeDragPosition`?